packages/code-studio/src/index.tsx

```diff
@@ -5,9 +5,12 @@ import { Provider } from 'react-redux';
 import { LoadingOverlay, preloadTheme } from '@deephaven/components';
 import { ApiBootstrap } from '@deephaven/jsapi-bootstrap';
 import { store } from '@deephaven/redux';
-import logInit from './log/LogInit';
+import { logInit } from '@deephaven/log';
 
-logInit();
+logInit(
+  parseInt(import.meta.env.VITE_LOG_LEVEL ?? '', 10),
```

Comment:
Is it ok that `''` parses to `NaN`?

Reply:
This is existing code moved from `LogInit` - https://github.com/deephaven/web-client-ui/pull/2279/files#diff-8f738bb62eda99bdb9b608feb9a011431c9daef91788b6ad56166ff4d950ca9eL15
I don't know if this was the original intent, but `NaN` here results in a warning - 
```
if (!Number.isFinite(level)) {
      console.warn(
        `Expected a number for log level. Received: ${level}. Ignoring`
      );
      return;
    }
```